components/FeatureFeed/FeatureFeed.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box } from '../../ui-kit';
+
+import { FeatureProvider } from '../../providers';
+import {
+  HeroListFeature,
+  HorizontalCardListFeature,
+  VerticalCardListFeature,
+} from '../index';
```

Comment:
A few things:
- With the latest `main` branch pulled in you, can do do `import { Box } from 'ui-kit';`.
- Same with the `'../../providers'` becoming `'providers'` instead.
- You can just do `'../'` without the `index`.
- Can we move the `import { Box }` down after the `providers` import? I like to keep the third-party imports separate from our own code.